yjit/bindgen/src/main.rs

```diff
@@ -58,6 +61,10 @@ fn main() {
 
         .allowlist_type("ruby_method_ids")
 
+        .allowlist_function("rb_callable_method_entry")
+        .constified_enum_module("rb_method_visibility_t") // These aren't generating bindings. Why?
+        .constified_enum_module("rb_method_type_t")
```

Comment:
Usually it's because it can't find the definition of the type (missing header?). It is a bummer that it seems to silently fail though. Might be a good idea to open an issue on bindgen because the silent failure seems like a design flaw. Unless there's some kind of verbose mode that would help us diagnose the issue.

Reply:
In this case, it's definitely not missing the header. Yeah, the silent failures aren't my favourite. I'll file an issue and look for a verbose mode.

Reply:
For the record: not sure how much verbose mode will help us since we run with a lot of warnings anyway (e.g. many, many unused-func warnings.) But I'm not seeing a warning for "func is on the allowlist but I didn't find it," and Googling isn't seeing much that's promising.

I may be missing something. Though I suppose trying to *use* these bindings will immediately get us an error if they don't exist, so there's that.

Reply:
Hopefully we can get the bindgen people to pay attention to this issue. Seems like a big flaw. I think they've probably never tried to use/test their tool on a codebase as large as CRuby.